Consul/Configuration.cs

```diff
@@ -1023,8 +1024,8 @@ public class ServiceResolverEntry : IConfigurationEntry
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public string Filter { get; set; }
 
-        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
-        public bool OnlyPassing { get; set; }
+        //[JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        //public bool OnlyPassing { get; set; }
```

Comment:
Is it intentional change?

Reply:
Yep, the class structure is different from Golang code, and with OnlyPassing not commented out it causes an error. I was still kinda testing things out, so I guess I should edit the structure to align with Golang code?
![image](https://github.com/user-attachments/assets/d6af5a1d-7bbd-4269-ab6c-3afd2c677448)
https://github.com/hashicorp/consul/blob/v1.9.17/api/config_entry_discoverychain.go#L136

Reply:
yes, please update this struct to align it with the golang code.